build/app/app.js

```
@@ -154,6 +155,8 @@ class VoiceIsolatePro {
     this.three = {};
 
     // v20: Initialize PipelineState (centralized 52-param state with pub/sub)
+    this.customPresets = JSON.parse(localStorage.getItem('vip_custom_presets') || '{}');
```

ESLint

[error] 'localStorage' is not defined.

(no-undef)